kubernetes-python/__main__.py

```diff
@@ -0,0 +1,16 @@
+import pulumi
+from pulumi_kubernetes.apps.v1 import Deployment
+
+app_labels = { "app": "nginx" }
+
+deployment = Deployment("nginx",
+    spec={
+        "selector": { "match_labels": app_labels },
+        "replicas": 1,
+        "template": {
+            "metadata": { "labels": app_labels },
+            "spec": { "containers": [{ "name": "nginx", "image": "nginx" }] }
+        }
+    })
+
+pulumi.export("name", deployment.metadata.apply(lambda m: m.name))
```

Comment:
You should be able to do:

```
pulumi.export("name", deployment.metadata.name)
```

This should just work - there's some sugar under the hood that does the `apply` for you.